app/src/main/java/com/example/vybe/Models/SocialSituation.java

```diff
@@ -0,0 +1,52 @@
+package com.example.vybe.Models;
+
+import androidx.annotation.NonNull;
+
+import java.util.ArrayList;
+
+public enum SocialSituation {
+    Alone("Alone"),
+    WithAnotherPerson("With one other person"),
+    WithSeveralPeople("With two or several people"),
+    WithCrowd("With a crowd"),
+    Same("Same");
```

Comment:
why do we need this?

Reply:
It's supposed to make it easier to work with SocialSituations

Reply:
sorry, I meant to ask if we need the option "same"

Reply:
same